app/domain/projects/services/class.projects.php

```diff
@@ -135,7 +135,7 @@ public function getUsersToNotify($projectId)
             //Only users that actually want to be notified and are active
             foreach ($users as $user) {
                 if ($user["notifications"] != 0 && strtolower($user["status"]) == 'a') {
-                    $to[] = $user["username"];
+                    $to[] = $user["id"];
```

Comment:
Did you test this thoroughly? 
I am trying to wrap my head around this and I know that the queue will check for email or ID. So that should be fine. Are there any other cases where we actually expect the email address instead of ID? 

I agree that we need to make sure the author check is correct which it is not right now (Great catch on that one!)

Reply:
Unfortunately, I did not get to the proper testing. I've only known the system for a few days, I don't have a general knowledge of what all needs to be tested.
However, in the IDE I went through places that use this method and all places should handle the ID correctly (I hope I'm not mistaken).
Please check on your side.
I just know that with this modification, my mails go correctly when I add/change tasks.
Thank you